app/angular/package.json

```diff
@@ -65,7 +65,7 @@
     "@angular/platform-browser-dynamic": ">=6.0.0",
     "@babel/core": "*",
     "rxjs": "^6.0.0",
-    "typescript": "^3.4.0",
+    "typescript": "^3.9.3",
```

Comment:
I'm not sure we should and need to bump this `peerDependencies` as it will create warnings when running `yarn/npm install` in a project using for instance Angular 6.

Reply:
We have `"typescript": "^3.9.3",` in root `package.json` so we should be already using 3.9.